### ISSUE TITLE
Increase fps

### ISSUE THREAD

**Issue by djnewty** [darsain/motio#16]
Hi,

Im wondering if there is any way to increase the fps above 60? I'm not really worried about performance :)

Ive looked all through the code and cant see where it is limited to 60. Any ideas where this is?

Cheers

**Comment by darsain** [comment]
It is limited by animation timing interface (of which requestAnimationFrame is part of), which is in sync with refresh rate of your monitor (most likely 60fps).

When you enter fps equal or higher than 60, all that happens is that motio's artificial render throttling (used for smaller fps) is turned off and the animation is rendered in every animation frame.

In other words, you can't render between frames.

You could maybe skip one sprite image, creating an illusion of higher fps (or faster animation), but that would be redundant, because why would you be creating a sprite where every 2nd frame is ignored? That's just wasting bandwidth.

Basically act like 60fps is the limit, because in practice it is :)

**Comment by djnewty** [comment]
Shame, seems like http://www.thoro.de/page/3dnp-introduction-en can spin faster than 60fps so might have to use that. Shame because I kinda like how motio works.
